.circleci/config.yml

```diff
@@ -346,46 +364,6 @@ jobs:
             - dist
             - builds
 
-  prep-build-beta:
-    executor: node-browsers-medium-plus
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - when:
-          condition:
-            not:
-              matches:
-                pattern: /^master$/
-                value: << pipeline.git.branch >>
-          steps:
-            - run:
-                name: build:dist
-                command: yarn build --build-type beta dist
```

Comment:
we might still want the beta build on branches other than the RC branch... so that other PRs can be tested on the beta build

What were your thoughts on removing this?

Reply:
I thought we would only need the beta on RC branch and we can test the feature branch with normal build. But if the beta is also required in non-rc i can bring it back ~

Reply:
bring it back now 🙏